commands/command_edit.cpp

```diff
@@ -1285,6 +1286,62 @@ void EditCylinder::undo()
 	redo();
 }
 
+EditSensors::EditSensors(int cylIndex, divecomputer *dcIn, std::vector<int> sensors)
+	: EditDivesBase(true), dc(dcIn)
+{
+	if (dives.empty())
+		return;
+
+	if (dives.size() == 1)
+		setText(Command::Base::tr("Edit sensors"));
+	else
+		// This shouldn't happen.
+		setText(Command::Base::tr("Edit sensors, only single dive computer"));
```

Comment:
Would be visible like this, but I guess it's a waste to translate it. Maybe just remove the whole if since it's always 1 dive now.